src/token-controls/styles/token-controls.scss

```
@@ -654,3 +654,184 @@
+ *
+ * The box's own border/radius/padding lives here rather than on `TokenSelector`'s trigger (which
+ * `.kb-token-control__row`'s shared rule styles for every other box field): Border's box wraps
+ * three siblings, not one, so the width trigger's own border is stripped down to nothing and only
+ * the outer box's edge shows.
+ */
+.kb-border-control__box {
+	display: flex;
+	align-items: center;
+	gap: 8px;
+	width: 100%;
+	// Tighter on the left than the `7px 10px` other box fields use — the reference seats the
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)